File: back-end/questoes-br/entity/Partida.py
from pydantic import BaseModel
from typing import Optional
from .Jogador import Jogador

class Partida(BaseModel):
    id_partida: Optional[int] = None
    id_jogador: int
    jogador: Jogador
    pontuacao: int = 0
    vidas: int = 0

File: back-end/questoes-back/entity/Partida.js
/**
 * 
 * @param {string} id_jogador 
 * @param {string} apelido_jogador 
 * @param {number} pontuacao pontuação do jogador durante o jogo
 * @param {number} vidas vidas do jogador
 * @param {Array <object>} historico_respostas histórico das respostas do jogador
 * @returns {object} partida partida criada
 */

export function criar_partida(id_partida, id_jogador, apelido_jogador, pontuacao, vidas, historico_respostas) {
    return {
        id: id_partida,
        id_jogador: id_jogador,
        apelido_jogador: apelido_jogador,
        pontuacao: pontuacao,
        vidas: vidas,
        historico_respostas: historico_respostas
    };
}